src/admin/admin.controller.ts

```diff
@@ -113,6 +115,21 @@ export class AdminController {
     return report;
   }
 
+  @ApiOkResponse({
+    description: 'Report has been successfully transferred',
+    type: TransferReportDto,
+  })
+  @Post('/reports/transfer')
+  async transferReport(
+    @Body() transferReportDto: TransferReportDto,
```

Comment:
Is it necessary to pass the entire `TransferReportDto`, or would just the `ReportID` suffice for this method?